src/vs/workbench/services/userDataProfile/browser/userDataProfileImportExportService.ts

```diff
@@ -271,36 +271,35 @@ export class UserDataProfileImportExportService extends Disposable implements IU
 				}
 				const message = localize('export success', "Profile '{0}' was exported successfully.", profile.name);
 				if (profileContentHandler.extensionId) {
-					const actions: string[] = [];
+					const buttons: IPromptButton<void>[] = [];
 					const link = this.productService.webUrl ? `${this.productService.webUrl}/${PROFILE_URL_AUTHORITY}/${id}/${saveResult.id}` : toUserDataProfileUri(`/${id}/${saveResult.id}`, this.productService).toString();
-					actions.push(localize('copy', "Copy Link"));
+					buttons.push({
+						label: localize({ key: 'copy', comment: ['&& denotes a mnemonic'] }, "&&Copy Link"),
+						run: () => this.clipboardService.writeText(link)
+					});
 					if (this.productService.webUrl) {
-						actions.push(localize('open', "Open Link"));
-					} else {
-						actions.push(localize('open in', "Open in {0}", profileContentHandler.name));
-					}
-					actions.push(localize('close', "Close"));
-					const result = await this.dialogService.show(
-						Severity.Info,
-						message,
-						actions,
-						{ cancelId: 2 }
-					);
-					switch (result.choice) {
-						case 0:
-							await this.clipboardService.writeText(link);
-							break;
-						case 1:
-							if (this.productService.webUrl) {
+						buttons.push({
+							label: localize({ key: 'open', comment: ['&& denotes a mnemonic'] }, "&&Open Link"),
+							run: async () => {
 								await this.openerService.open(link);
-							} else {
+							}
+						});
+					} else {
+						buttons.push({
+							label: localize({ key: 'open in', comment: ['&& denotes a mnemonic'] }, "&&Open in {0}", profileContentHandler.name),
+							run: async () => {
 								await this.openerService.open(saveResult.link.toString());
 							}
-							break;
-
+						});
 					}
+					await this.dialogService.prompt({
```

Comment:
@sandy081 fyi please check, this one is a bit more complex adoption of the new dialog api